boards/sifive/hifive1/Kconfig.hifive1

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Antmicro 2024 <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_HIFIVE1
+	select SOC_SIFIVE_FREEDOM_E310
```

Comment:
does not select the additional Kconfig for revb

Reply:
fixed